src/main/java/com/dongxuexidu/douban4j/provider/OAuthDoubanProvider.java

```diff
@@ -148,7 +148,7 @@ public AccessToken tradeAccessTokenWithCode (String code) throws DoubanException
       Map<String,String> params = new HashMap<String, String>();
       params.put("client_id", this.apiKey);
       params.put("client_secret", this.secretKey);
-      params.put("redirect_uri", DefaultConfigs.ACCESS_TOKEN_REDIRECT_URL);
+      params.put("redirect_uri",  this.redirectUrl);
```

Comment:
according to the playground.java process, i think tradeAccessTokenWithCode must be invoke after getGetCodeRedirectUrl, if null value appears, tradeAccessTokenWithCode will not be executed.

but still , adding default value for "redirectUrl" is much safer.